### PR TITLE
docs(README.md): update readme

### PR DESCRIPTION
**Pull request by kleberbaum**
- Add Motivation
- Add Get Up and Running in 5 Minutes
- Add How to Code
- Update How to Report a Bug or Request a Feature
- Update How to Contribute
- Remove Versioning
- Update Thanks
- Update Creators
- Update FAQs
- Update Copyright and License

**Checklist:**

 

-   [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
-   [x] Have you updated or added documentation for the change?
-   [ ] Have you tested your changes with successful results?

**Type of Changes:**

 

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Documentation (non-breaking change which adds documentation)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

-

**What is the new behavior (if this is a feature change)?**

-

**Other information**:

-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,61 +71,273 @@ ERP integration **shouldn not** force developer to cut corners.
...
+| Feature                       | Shipped | Almost There | We're Writing the Code | Investigating |
+|-------------------------------|:---:|:---:|:---:|:---:|
+| `IndexField`                  | ✅️ |  |  |  |
+| `RichTextField`               | ✅️ |  |  |  |
+| `Email Support`               | ✅️ |  |  |  |
+| `Fixed parent for IndexField` | ✅️ |  |  |  |
+| `TextField`                   | ✅️ |  |  |  |
+| `Dynamic Routes`              | ✅️ |  |  |  |
+| `DocumentField`               |  | ✅️ |  |  |
+| `MediaField`                  |  | ✅️ |  |  |
```
Comment (schettn) [inline README.md:94]:
```suggestion
| `ImageField`                  |  | ✅️ |  |  |
```

File: README.md [hunk 2]
```diff
@@ -52,61 +71,273 @@ ERP integration **shouldn not** force developer to cut corners.
...
+### Roadmap
+| Feature                       | Shipped | Almost There | We're Writing the Code | Investigating |
+|-------------------------------|:---:|:---:|:---:|:---:|
+| `IndexField`                  | ✅️ |  |  |  |
+| `RichTextField`               | ✅️ |  |  |  |
+| `Email Support`               | ✅️ |  |  |  |
+| `Fixed parent for IndexField` | ✅️ |  |  |  |
+| `TextField`                   | ✅️ |  |  |  |
+| `Dynamic Routes`              | ✅️ |  |  |  |
+| `DocumentField`               |  | ✅️ |  |  |
```
Comment (schettn) [inline README.md:93]:
```suggestion
| `PdfField`               |  | ✅️ |  |  |
```

**Review by schettn — CHANGES_REQUESTED** [review]
have a look

**Review by kleberbaum — COMMENTED** [review]
Should be ok to merge.

**Review by schettn — APPROVED** [review]
lgtm

**Pull request merged**